.github/workflows/main.yml

```diff
@@ -24,9 +24,9 @@ jobs:
           fetch-depth: 50
 
       - name: Check changelog
-        # Skip for pmem/pmdk/master and stable-* branches
+        # Skip for daos-stack/pmdk/master and stable-* branches
         if: |
-          !(github.repository == 'pmem/pmdk' &&
```

Comment:
.

Reply:
I thinkl we should add here this:
`${{ github.event.repository.fork == false }}`
instead of `github.repository == 'pmem/pmdk'`
main.yml should run only on main repo (not on fork)

Reply:
@janekmi What do you think about co... suggestion:
```
on:
  schedule:
    - cron: "0 0 * * *"

jobs:
  scheduled-job:
    runs-on: ubuntu-latest
    steps:
      - name: Check for secret
        run: |
          if [ -z "${{ secrets.MAIN_REPO_SECRET }}" ]; then
            echo "Secret not found. Stopping workflow."
            exit 0
          fi
      - name: Run the scheduled job
        run: echo "This is the main repository. Proceeding with the job."

```

Reply:
I proposed a combination of the solution used in the docker_rebuild.yml file.
I don't know if introducing a new secret will be beneficial. @janekmi  what do you think?

Reply:
Discussed offline.